buildenv/jenkins/common/pipeline-functions.groovy

```diff
@@ -661,6 +662,25 @@ def retry_and_delay(command, numRetries = 3, waitTime = 60, units = 'SECONDS') {
 }
 
 def setup_pull_request() {
+    // Parse Github trigger comments
+    // For example:
+    /* This is great
+       Jenkins test extended <platform>
+       It was tested*/
+
+    def PARSED_BY_NEWLINE_COMMENT = params.ghprbCommentBody.split(/\\r?\\n/)
+    for (COMMENT in PARSED_BY_NEWLINE_COMMENT) {
+        echo "'${COMMENT}'"
```

Comment:
I would recommend when you are echoing info either for debug or permanent, you label what it is you are echoing, that way you know what is being printed. Echoing only the variable will make it hard to debug when it is an unexpected value like an empty string. However, if your echo includes either the variable name or some description it will be easier to debug.
For example
`echo "Trigger Comment:'${COMMENT}'"`
If COMMENT is empty you will at least have `Trigger Comment:''`